app/models/admin/FeatureFlag.scala

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package models.admin
+
 
 //import uk.gov.hmrc.mongoFeatureToggles.model.FeatureFlagName
 //
```

Comment:
This whole class is commented out, please remove

Reply:
Done